nptdms/tdms.py

```diff
@@ -23,9 +23,26 @@
 from nptdms import types
 
 
+# log level
+log_level = logging.WARNING
+
+# create logger
 log = logging.getLogger(__name__)
-logging.basicConfig()
-log.setLevel(logging.WARNING)
+log.setLevel(log_level)
+
+# create console handler and set level to debug
```

Comment:
Minor point but the comment doesn't match the code (level is not debug). The comment would probably be unnecessary if the console handler was named `console_handler`, and a lot of the comments below are also redundant.